### PR TITLE
DSDEEPB-2266: Passthrough for /inputsOutputs endpoint

### PR DESCRIPTION
**Pull request by MattPutnam**

### CODE REVIEW

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -136,6 +136,38 @@ paths:
...
+      produces:
+        - application/json
+      parameters:
+        - name: methodName
+          description: name of Method to look up
+          required: true
+          type: string
+          in: body
+          schema:
+            $ref: MethodID
```
Comment (davidangb) [inline src/main/resources/swagger/api-docs.yaml]: not your doing originally, but go down to the definition for MethodId around line 2142, and add default: 0 to the methodVersion. Without the default value, swagger won't show this key when pre-filling the payload. (with methodVersion missing from the payload, swagger returns no response; that was my problem)

**Comment by davidangb** [comment]
Looks good, with one minor swagger change please. Feel free to make the change and merge yourself, or make the change and pass back to me if you'd rather.

**Comment by MattPutnam** [comment]
@davidangb Can you do a quick check on the swagger?  I'm having trouble getting it to generate.

**Comment by davidangb** [comment]
:+1:

**Pull request merged**